### PR TITLE
Update resiliency docs

### PR DESCRIPTION
**Pull request by ywelsch**
Adds clarifications about Jepsen tests and new section on issues with versioning.

### CODE REVIEW

File: docs/resiliency/index.asciidoc [hunk 1]
```diff
@@ -102,6 +102,19 @@ space.  The following issues have been identified:
 
 Other safeguards are tracked in the meta-issue {GIT}11511[#11511].
 
+[float]
+=== The _version field may not uniquely identify document content during a network partition (STATUS: ONGOING)
+
+When a primary is partitioned away from the cluster there is a short period of time until it detects this. During that time it will continue
+indexing writes locally, thereby updated document versions. When it tries replicating the operation, however, it will discover that it is
```
Comment (jasontedor) [inline docs/resiliency/index.asciidoc]: Typo: `updated` -> `updating`

**Comment by jasontedor** [comment]
LGTM.

File: docs/resiliency/index.asciidoc (hunk 1 above)
Comment (clintongormley) [inline docs/resiliency/index.asciidoc]: indexing -> to index

Comment (clintongormley) [inline docs/resiliency/index.asciidoc]: replicating -> to replicate

File: docs/resiliency/index.asciidoc [hunk 2]
```diff
@@ -102,6 +102,19 @@ space.  The following issues have been identified:
...
+partitioned away. It won't acknowledge the write and wait until the partition is resolved to negotiate with the master on how to proceed.
```
Comment (clintongormley) [inline docs/resiliency/index.asciidoc]: is partitioned -> has been partitioned

Comment (clintongormley) [inline docs/resiliency/index.asciidoc]: write and wait -> write and will wait

File: docs/resiliency/index.asciidoc [hunk 3]
```diff
@@ -102,6 +102,19 @@ space.  The following issues have been identified:
...
+The master will decide to either fail any replicas which failed to index the operations on the primary or tell the primary that it has to
+step down because a new primary was chosen in the meantime. Since the old primary has already written documents, clients may have read from
```
Comment (clintongormley) [inline docs/resiliency/index.asciidoc]: was -> has been chosen

Comment (clintongormley) [inline docs/resiliency/index.asciidoc]: may have -> may already have

**Comment by clintongormley** [comment]
Minor grammar changes, but LGTM

**Comment by ywelsch** [comment]
Thanks @jasontedor @clintongormley.

**Pull request merged**